src/allotropy/parsers/lines_reader.py

```diff
@@ -36,74 +38,90 @@ def current_line_exists(self) -> bool:
     def get(self) -> Optional[str]:
         return self.lines[self.current_line] if self.current_line_exists() else None
 
-    def match(self, pattern: str) -> bool:
+    def match(self, match_pat: str) -> bool:
```

Comment:
If you're going to rename the parameters, could you spell out `pattern` rather than `pat`?

Reply:
I chose `match_pat` to be consistent with e.g. `pop_if_match` and `drop_until` further down in the same file - `pattern` seemed to be inconsistent (?).